mdpow/tests/test_automated_dihedral_analysis.py

```diff
@@ -96,17 +101,33 @@ def gen_data(self, SM25_tmp_dir):
                                 'H29', 'H30', 'H31', 'H32', 'H33', 'H34', 'H35'], dtype=object)
 
     # pre-padding dihedral group statistics
-    DG_O1C2N3S4_mean = -0.13089566578887002
-    DG_O1C2N3S4_var = 0.03473127346296412
+    #DG_O1C2N3S4_mean = -0.13089566578887002
+    #DG_O1C2N3S4_var = 0.03473127346296412
 
+    # new 2-6-23
     # pre-padding dihedral group statistics
-    DG_C13141520_mean = 90.04076626959608
-    DG_C13141520_var = 0.8814931303534448
+    DG_O1C2N3S4_mean = -0.5933808752787115
+    DG_O1C2N3S4_var = 0.031351024457919485
+
+    # pre-padding dihedral group statistics
+    #DG_C13141520_mean = 90.04076626959608
+    #DG_C13141520_var = 0.8814931303534448
+
+    # new 2-6-23
```

Comment:
Don't put new/old dates etc in here. It's not going to be helpful outside the narrow context of this PR.

The code needs to stand on its own.